zanata-war/src/main/webapp/WEB-INF/layout/project/settings-tab.xhtml

```diff
@@ -203,21 +210,39 @@
       #{msgs['jsf.project.WebHooks']}
   </h2>
   <h:form  id="settings-webhooks-list" onclick="return false">
+
+    <h3 class="heading--secondary">#{msgs['jsf.project.WebHooks.secretKey']}</h3>
+    <div class="form__item">
+      <div class="form--inline">
+        <input type="text" id="secretKeyInput" autocomplete="off" maxlength="255"
+          placeholder="#{msgs['jsf.project.WebHooks.secretKey.placeholder']}"
+          class="form--inline__input" value="#{projectHome.instance.webhookKey}"/>
+
+        <div class="form--inline__suffix">
+          <button id="add-webhook-key" class="form--inline__addon button--primary button loader"
+            onclick="addWebhookSecretKey(jQuery('#secretKeyInput').val())">
+            <span class="loader__label">#{msgs['jsf.project.AddWebhookSecretKey']}</span>
+          </button>
```

Comment:
Is there an actual button for adding the key?  Github just has an optional "Secret" field when creating or updating a Webhook.

It might be good to link to a page (eg on docs.zanata.org) which explains our Webhook authentication scheme.